src/AddTodoForm.js

```diff
@@ -1,43 +1,30 @@
-import React from 'react';
-// import * as reactDom from 'react-dom'
-import { useState } from "react";
-
-
-function AddTodoForm ({onAddTodo}) {
-
-const [todoTitle, setTodoTitle] = useState('');
-
-const handleTitleChange = (event) => {
-     const newTodoTitle = event.target.value
-     setTodoTitle(newTodoTitle)
-}
-
- const handleAddTodo = (event) => {
- event.preventDefault();
- //if todoTile === '' I don't want it to return anything
- if(todoTitle.trim().length === 0) return 
-      //   let onAddTodo;
-  onAddTodo({title: todoTitle, id: Date.now()}) // won't work without the props
-        setTodoTitle('')
-    }
-
-   return (
-    <div>
-    <form onSubmit={handleAddTodo} >
-  
-      <label htmlFor="todoTitle">Title: </label>
-      <input id='todoTitle' value={todoTitle} type="text" name='title' onChange={handleTitleChange} />
-      <button type='submit'>Submit</button>
-    
-      {/* <span>{todoTitle}</span> */}
-      
+import React, { useState } from "react";
+import InputWithLabel from "./InputWithLabel";
+
+const AddTodoForm = ({ onAddTodo }) => {
+  const [todoTitle, setTodoTitle] = useState("");
+
+  const handleTitleChange = (e) => {
+    setTodoTitle(e.target.value);
+  };
+
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    if (todoTitle.trim().length === 0) return;
+
+    onAddTodo({ title: todoTitle, id: Date.now() }); // won't work without the props
+    setTodoTitle("");
+  };
+
+  return (
+    <form onSubmit={handleAddTodo}>
+      <InputWithLabel
+        value={todoTitle}
+        onChange={handleTitleChange}
```

Comment:
I would still recommend giving these more verbose names per the suggestion. It gets more confusing in the `InputWithLabel` component because the `<input>` has props like `value={value}` and `onChange={onChange}` which aren't very descriptive. Imagine not working on this code for month and having to try to remember what those things mean, or if someone else has to work on your code. Without meaningful names you'd have to track down the parent component to figure out what these are referring to.